src/toolkit/position.cycpp.h

```diff
@@ -0,0 +1,32 @@
+
+// This includes the required header to add geographic coordinates to a
+// archetypes.
```

Comment:
I thinks we need a longer message here that explains how it all works, since it is a brand new concept,

Also, can we use one of the archetypes that’s in `cyclus/cyclus` to demonstrate?